hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/snapshot/TestRocksDbPersistentMap.java

```diff
@@ -131,4 +147,93 @@ public void testRocksDBPersistentMap() throws IOException, RocksDBException {
       }
     }
   }
+
+  /**
+   * Test cases for testRocksDBPersistentMapIterator.
+   */
+  private static Stream<Arguments> testRocksDBPersistentMapIteratorCases() {
```

Comment:
It is not test, more of testcases. Please update the comment as well.
```suggestion
  private static Stream<Arguments> rocksDbPersistentMapIteratorTestCases() {
```

Reply:
done

Reply:
Comment should be

```
* Test cases for RocksDbPersistentMapIterator.
```